p/scripts/main.js

```diff
@@ -1579,11 +1587,9 @@ init_beforeDOM();	//Can be called before DOM is fully loaded
 if (document.readyState && document.readyState !== 'loading') {
 	init_afterDOM();
 } else {
-	document.addEventListener('DOMContentLoaded', function () {
-			if (window.console) {
-				console.log('FreshRSS waiting for DOMContentLoaded…');
```

Comment:
Minor: Moved the log message outside of the callback so that it is printed immediately